src/Mod/Arch/ArchStructure.py

```diff
@@ -183,6 +183,76 @@ def placeAlongEdge(p1,p2,horizontal=False):
     return pl
 
 
+class _CommandStructuresFromSelection:
```

Comment:
Do not add classes with an `_underscore` name, because they will not be parsed automatically by tools such as Doxygen and Sphinx when generating documentation. It is like this in older code, but this is fundamentally a mistake that we should avoid in newer code.

Names with underscore should be reserved for completely meaningless classes, functions and variables that we want hidden. Major classes, such as scripted objects and Gui Commands, are not in this group.